src/directive/user.ts

```diff
@@ -41,6 +42,10 @@ export const deleteUser = async (
       await entityManager.getRepository(SourceDisplay).delete({ userId });
       await entityManager.getRepository(SourceRequest).delete({ userId });
       await entityManager.getRepository(Upvote).delete({ userId });
+      // Manually set shared post to 404 dummy user
+      await entityManager
```

Comment:
I think we should apply it on every post that is no `ArticlePost`.
Maybe we should apply it also for comments?

Reply:
Good point, can do that actually :) 
And also for comments seems like a good way to preserve conversations.